src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -321,14 +322,21 @@ private async Task PublishFunctionApp(Site functionApp, GitIgnoreParser ignorePa
                 await Task.Delay(TimeSpan.FromSeconds(5));
                 await SyncTriggers(functionApp);
             }
+
+            // Linux Elastic Premium functions take longer to deploy. In order to view the functions properly,
+            // we need to wait for 15 extra seconds
```

Comment:
how did you land at 15 seconds? From what I've seen, this can be significantly longer.

Reply:
Through trial and error, 15 seconds seemed to work for all of my cases. However, my functions were very small, and if that's not a constant time, I think we should not show functions at all for Elastic Premium. What's your thought?

Reply:
I think we shouldn't show it if there is a chance (which there is) they will frequently be outdated. 

We need a reliable way to know that the runtime restarted and that we're getting the list from the new runtime, which as far as I know there isn't one for App Service Linux.

Reply:
Fair! I updated the PR.

Reply:
I'll be also good to communicate that to the premium PMs

Reply:
I told @dariagrigoriu about this! Will let others know